apps/ledger-live-mobile/src/components/TabBar/TransferDrawer.tsx

```diff
@@ -71,11 +72,13 @@ export default function TransferDrawer({ onClose }: Omit<ModalProps, "isRequesti
     [navigation, onClose],
   );
   const onNavigateRecover = useCallback(() => {
-    if (learnMoreURI) {
+    if (quickAccessURI) {
+      Linking.canOpenURL(quickAccessURI).then(() => Linking.openURL(quickAccessURI));
+    } else if (learnMoreURI) {
```

Comment:
Not sure about that 

`quickAccessURI` will be defined every time no? so `learnMoreURI` is unreachable

Reply:
I want to priorize quickAccessURI first. 
If it's not present, it take learnMoreURI. 
It's just to prevent the update on prod featureflags

Do you think it's better to put only QuickAccessURI and wait the update of featureFlags prod ?

Reply:
https://github.com/LedgerHQ/ledger-live/pull/4884/commits/e65f0e505e5ee78adaf177b0a9cefde8eaebb2fb

Reply:
Ok so now you're using only QuickAction :) 

LGTM on the dev side !